demo/scripts/controls/sidePane/apiPlayground/vtable/VTablePane.tsx

```diff
@@ -1,364 +1,776 @@
-            editor.focus();
-            editor.select(td, PositionType.Begin);
-        });
-        this.createVTable();
-    };
-}
-
-function createTableFormat(
-    bgColorEven?: string,
-    bgColorOdd?: string,
-    topBorder?: string,
-    bottomBorder?: string,
-    verticalBorder?: string
-): TableFormat {
-    return {
-        bgColorEven: bgColorEven,
-        bgColorOdd: bgColorOdd,
-        topBorderColor: topBorder,
-        bottomBorderColor: bottomBorder,
-        verticalBorderColor: verticalBorder,
-    };
-}
-
-function CustomizeFormatRow(props: { text: string; inputRef: React.RefObject<HTMLInputElement> }) {
-    const [isColorPickerShowing, setIsColorPickerShowing] = React.useState(false);
-    const onShowColorPicker = React.useCallback(() => {
-        setIsColorPickerShowing(!isColorPickerShowing);
-    }, [isColorPickerShowing]);
-    const setColor = React.useCallback((color: Color) => {
-        props.inputRef.current.value = color.hex().toString();
-    }, []);
-    let color: Color;
-    try {
-        color = Color(props.inputRef.current.value);
-    } catch {
-        color = Color('white');
-    }
-    return (
-        <>
-            <tr>
-                <td className={styles.label}>
-                    <button onClick={onShowColorPicker}>{props.text}</button>
-                </td>
-                <td>
-                    <input type="text" ref={props.inputRef} />
-                </td>
-            </tr>
-            {isColorPickerShowing && (
-                <tr>
-                    <td colSpan={2}>
-                        <ColorPicker initColor={color} onSelect={setColor} />
-                    </td>
-                </tr>
-            )}
-        </>
-    );
-}
+import * as Color from 'color';
+import * as React from 'react';
+import ApiPaneProps from '../ApiPaneProps';
+import ColorPicker from '../../../colorPicker/ColorPicker';
+import { formatTable } from 'roosterjs-editor-api';
+import { getTableFormatInfo, getTagOfNode, saveTableInfo, VTable } from 'roosterjs-editor-dom';
+import {
+    IEditor,
+    PositionType,
+    TableBorderFormat,
+    TableFormat,
+    TableOperation,
+    VCell,
+} from 'roosterjs-editor-types';
+
+const PREDEFINED_STYLES: Record<string, (color?: string, lightColor?: string) => TableFormat> = {
+    GRID_WITHOUT_BORDER: (color, lightColor) =>
+        createTableFormat(
+            null /**bgColor */,
+            color /**topBorder */,
+            color /**bottomBorder */,
+            color /** verticalColors*/,
+            true /** bandedRows */,
+            false /** bandedColumns */,
+            true /** headerRow */,
+            false /** firstColumn */,
+            TableBorderFormat.onlyMiddleBorders /** tableBorderFormat */,
+            lightColor /** bgColorEven */,
+            null /** bgColorOdd */,
+            null /** bgColumnColorOdd */,
+            null /** bgColumnColorEven */,
+            null /** headerRowColor */

```

Comment:
I would always add values here. this values wont be applied to the table format because the banded columns is false